03week/towersOfHanoi.js

```diff
@@ -19,23 +19,80 @@ function printStacks() {
   console.log("c: " + stacks.c);
 }
 
```

Comment:
missing a readme code plan file, but will take the comments a part of it.
missing additional 3 tests.